modules/migration-migrating-applications-api.adoc

```diff
@@ -242,16 +242,37 @@ spec:
     name: <migstorage_ref> <3>
     namespace: openshift-migration
   namespaces:
-    - <application_namespace> <4>
+    - <source_namespace_1> <4>
+    - <source_namespace_2>
+    - <source_namespace_3>:<destination_namespace> <5>
   srcMigClusterRef:
-    name: <remote_cluster_ref> <5>
+    name: <remote_cluster_ref> <6>
     namespace: openshift-migration
 ----
 <1> Direct image migration is enabled if `false`.
 <2> Direct volume migration is enabled if `false`.
 <3> Specify the name of the `MigStorage` CR instance.
-<4> Specify one or more namespaces to be migrated.
-<5> Specify the name of the source cluster `MigCluster` instance.
+<4> Specify one or more source namespaces. If you specify only the source namespace, the destination namespace is the same.
+<5> Specify a destination namespace if it is different from the source namespace.
+<6> Specify the name of the source cluster `MigCluster` instance.
++
+[IMPORTANT]
```

Comment:
@XinRedhat Here is the warning about duplicating or merging namespaces.

Reply:
@apinnick got it . then it looks good to me